cassandra-sink/src/main/java/org/springframework/xd/sink/cassandra/CassandraSinkOptionsMetaData.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.sink.cassandra;
+
+import javax.validation.constraints.AssertFalse;
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.cassandra.core.ConsistencyLevel;
+import org.springframework.cassandra.core.RetryPolicy;
+import org.springframework.integration.cassandra.outbound.CassandraMessageHandler;
+import org.springframework.util.StringUtils;
+import org.springframework.xd.module.options.mixin.CassandraConnectionMixin;
+import org.springframework.xd.module.options.spi.Mixin;
+import org.springframework.xd.module.options.spi.ModuleOption;
+
+/**
+ * @author Artem Bilan
+ */
+@Mixin(CassandraConnectionMixin.class)
+public class CassandraSinkOptionsMetaData {
+
+	private ConsistencyLevel consistencyLevel;
+
+	private RetryPolicy retryPolicy;
+
+	private Integer ttl;
```

Comment:
No, I can't. See the logic around this variable.

In this class below:
```
if (this.consistencyLevel != null || this.retryPolicy  != null || this.ttl != null) {
			cassandraMessageHandler.setWriteOptions(new WriteOptions(this.consistencyLevel, this.retryPolicy, this.ttl));
		}
``` 
And further in the `CqlTemplate`
```
if (options.getTtl() != null) {
	q.using(QueryBuilder.ttl(options.getTtl()));
}
```
where `QueryBuilder`:
```
 if (ttl < 0)
        throw new IllegalArgumentException("Invalid ttl, must be positive");

return new Using.WithValue("TTL", ttl);
```

So, I really don't like to overhead each query with that redundant extract `TTL` option for `0`.